Tests/Realm.Tests/Database/TestObjects.cs

```diff
@@ -1141,6 +1143,10 @@ public partial class LinksObject : TestRealmObject
 
         public IDictionary<string, LinksObject?> Dictionary { get; } = null!;
 
+        public LinksObject()
```

Comment:
Why do we need this ctor?

Reply:
It is necessary when this is a woven class. I'll put preprocessor directives around it to make it clearer.

Reply:
Ugh... might need to suppress the id warning then.

Reply:
Yes, I'll fix that too